### ISSUE TITLE
Clicking previous sidebar nav tab is not possible while a new sidebar content is loading

### ISSUE THREAD

**Issue by Vainonen** [NatLibFi/Skosmos#1361]
## At which URL did you encounter the problem?
When clicking a sidebar tab (URL ending with index, hierarchy, groups, new), sidebar content starts to load, but the previous tab link is inactive until loading is finished.

## What steps will reproduce the problem?
1. This issue is prevalent, If the changes in https://github.com/NatLibFi/Skosmos/pull/1360 PR are taken into use.
2. Click a sidebar nav tab (e. g. http://finto.fi/yso/fi/index) that is not currently active
3. When sidebar content is loading, try to click the previous tab link, which should be inactive during the content loading

## What is the expected output? What do you see instead?
Is this behavior wanted? Making current active sidebar tab link inactive to prevent unnecessary requesting of same sidebar content . The user might click a tab by accident and might want to return to the previous tab instantly. On the other hand, is it better to reduce user's possibilities to make sidebar content requests without limitations?

## What browser did you use? (eg. Firefox, Chrome, Safari, Internet explorer)
Firefox, Chrome

**Comment by osma** [comment]
I don't think this is relevant for Skosmos 3 development, since it has a new implementation of the sidebar. Closing the issue, please reopen if still relevant.
